### PR TITLE
fixed bug where no imageurl is passed, and also #70

### LINKED ISSUE #70: renderFooter example
I have my ImageViewer as 

```
                     <ImageViewer
                            imageUrls={fullImagesURLs}
                            index={this.state.curIndex}
                            renderFooter = { () => {
                                return <Text>FooterText</Text>
                            }}
                        />
```

But this Footer Text is not visible in full screen mode. I referred to [#51](https://github.com/ascoders/react-native-image-viewer/issues/51)  but can't figure out what's wrong in above example.

### PR DESCRIPTION
**Pull request by ShayanJavadi**
This fixes the lib throwing an error for panHandlers and originSizeKb when no imageUrls are passed.

it also removes a unused style object from the Y axis animated view

it also fixes #70 by adding a wrapper around the footer element that ensures that it says at the bottom. I added a prop called `footerContainerStyle`which lets you change the style of this new wrapper.

here's the fix for renderFooter in action:
screen shot 2018-03-19 at 4 49 35 pm

### CODE REVIEW

File: src/image-viewer.component.tsx [hunk 1]
```diff
@@ -616,11 +616,13 @@ export default class ImageViewer extends React.Component<Props, State> {
...
       }
     })
+    const panResponderYPanHandlers = this.state.panResponderY ?
+    this.state.panResponderY.panHandlers :
+    {}
 
     return (
       <Animated.View
-        style={{ zIndex: 9999 }}
-        {...this.state.panResponderY.panHandlers}
```
Comment (ascoders) [inline src/image-viewer.component.tsx]: This line is added by #153 , and i'll to restructure it, you just delete it, this is just right.

**Comment by ascoders** [comment]
Thank you so mush, but it a bit conflict with #153, and i wll restructure it.

File: src/image-viewer.component.tsx (hunk 1 above)
Comment (ShayanJavadi) [reply src/image-viewer.component.tsx]: I didn’t delete the line. I’m checking if panResponderY for undefined so no error gets thrown :)

Comment (ascoders) [reply src/image-viewer.component.tsx]: Well, i'm sorry to misread it.

**Pull request merged**